tokenizer.go

```diff
@@ -1,29 +1,30 @@
 package tools
 
-import "gopkg.in/bblfsh/sdk.v1/uast"
+import (
+	"github.com/bblfsh/go-client/v4/tools"
+	"github.com/bblfsh/sdk/v3/uast"
+	"github.com/bblfsh/sdk/v3/uast/nodes"
+)
 
+// Tokenizer sub-command outputs every token to STDOUT.
```

Comment:
I imagine this won't work as one may expect. Tokens are missing for some structures.